### ISSUE TITLE
"File operation blocked by the system" 

### ISSUE THREAD

**Issue by Noxave** [dodgingpixels/Optic#42]
When i try to delete pictures from folders that are in the sd card the app give me the message "File operation blocked by the system"  Android 7.1.2 Carbon ROM

**Comment by kruleucc** [comment]
This got me thinking too.
Huawei P10 Lite, stock 7.0

**Comment by staber** [comment]
Can you try to move a file to or from the SD Card first? This will show a new dialog to accept permission for writing to the SD Card. After that, try deleting again.

**Comment by kruleucc** [comment]
Tried moving it to sd card, says this: https://postimg.org/image/4pivogl11/

I have choosen the root of sd card as it told me to.

**Comment by Noxave** [comment]
Got exactly the same toast notification as krux3r, "there is no write acces to this directory, please choose a different one"

**Comment by kruleucc** [comment]
@LittleSnevil I'm having these kind of problems with all the galleryes from Play Store, only Piktures can delete or move photos that are on Sd card. Strange...

**Comment by Noxave** [comment]
@krux3r with Simple Gallery i don't have this problem.  But if you say that with _**all**_ galleries you're experiencing this, that is strange.. hope this problem got solve in Optic, i really love this project.

**Comment by kruleucc** [comment]
@LittleSnevil yeah, my excuses, Simple Gallery works ok, just tried. I select the root of the sd card and afterwards I am able to delete or kove anything on SD card.
I agree, Optic is such a nice app:blush:

**Comment by kruleucc** [comment]
@staber any news? :)

**Comment by staber** [comment]
@krux3r I see what the issue is, it should be fixed for the next update.

**Comment by kruleucc** [comment]
@staber nice to hear that, I'm hyped haha :)

**Comment by kruleucc** [comment]
@staber any ETAs for the update? Sorry for being annoying but I can't wait for another update.

**Comment by staber** [comment]
Sorry about that. Got caught up in a Kotlin conversion. Hopefully by the end of the week.

**Comment by kruleucc** [comment]
All good man. Looking forward to it :)

**Comment by staber** [comment]
Should be fixed in v0.6.0 update. Let me know if not and I'll re-open. (Sorry, don't have many SD devices to test with)

**Comment by Noxave** [comment]
Deleting pics seems to be fixed. But when I try to moves between SD Card and internal storage, keeps saying "there is no write acces to this directory, please choose a different one"

**Comment by staber** [comment]
Was moving working before this update?
Is it happening in both directions? Internal -> External & External -> Internal

**Comment by kruleucc** [comment]
Deleting is working great now.
About moving, both internal and external shows what @LittleSnevil said.

**Comment by Noxave** [comment]
This is happening to me now, and it's very strange. 
When using Simple Gallery, External -> Internal works. Internal -> External don't let me. Tells me something about magane_documents.

BUT with Optic, External -> internal works great. The problem is now Internal -> External. When I move the file, it simply disappear. I thought it was deleting my files but it's sending my files to the Send folder of WhatsApp 🤨🤔 Wich is usually hidden.

**Comment by kruleucc** [comment]
For me, external -> internal works, but internal -> external doesn't. It shows that directory error.

Also, please include the Croatian translations as I completed them @staber.

**Comment by fox1047** [comment]
Same problem. External -> Internal works, but Internal -> External don't work.
Device: Asus ZB602KL
OS: Android 8.1.0
